### PR TITLE
Iakow/movie-seat-booking

### PR DESCRIPTION
**Pull request by Iakow**
[Demo](https://iakow.github.io/kottans-movie/)
[Code](https://github.com/kottans/frontend-2021-classroom/tree/iakow/movie-seat-booking/submissions/iakow)

### CODE REVIEW

File: submissions/iakow/index.html [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+  </head>
+  <body>
+    <div class="booking-popup">
+      <header>
+        <nav><button class="btn-back" ></button></nav>
+        <h2>Select seats</h2>
+      </header>
+
+      <section class="movie-info">
+        <h3 class="movie-name">"Simplicity"</h3>
```
Comment (A-Ostrovnyy) [inline submissions/iakow/index.html]: For main page title use `h1` tag.

File: submissions/iakow/index.html [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+        <time class="movie-date" datetime="2001-05-15 19:00">
+          15 Feb - 20:20
+        </time>
```
Comment (A-Ostrovnyy) [inline submissions/iakow/index.html:20]: 🔥

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comment below!

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Don't forget to update your demo, also, remove files not for review(images, etc)

**Comment by Iakow** [comment]
> Don't forget to update your demo, also, remove files not for review(images, etc)

Sorry, it seemed to me that Ivan expressed uncertainty about this at the first zoom session. In any case - done)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**